api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -5696,7 +5696,7 @@
             "x-kubernetes-patch-strategy": "merge"
           },
           "containerStatuses": {
-            "description": "The list has one entry per container in the manifest. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#pod-and-container-status",
+            "description": "Statuses of containers in this pod. The list should not contain entries with the same container name, and should only contain statuses for containers listed in pod spec. However this is not enforced. If status for not-existing container is present in the list, the behavior of various Kubernetes components is not defined and those statuses might be ignored. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#pod-and-container-status",
```

Comment:
`The list should not contain entries with the same container name` is a little clunky to me maybe

`Each container in the pod should have at most one status in this list, and all statuses should be for containers in the pod.`

also maybe `a non-existent` instead of `not-existing`

Reply:
thanks, updated